lib/utils/aws/generate.go

```diff
@@ -0,0 +1,20 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package aws
+
+//go:generate go run ./internal/region/codegen.go
+//go:generate gofmt -w -s .
```

Comment:
should we put the `go:generate` comment in `region.go`?

for `codegen.go`, is it possible to put outside of `lib` (maybe `./build.assets/tooling`)?

Reply:
We could place it. I just wanted to ensure we generate this using the same version we're using at Teleport; if we place it on the tooling go module, we must keep it in sync and update it instead to get any new region. Still, I don't have any strong opinion on this.